Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "alvarium-sdk-rust"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = ["rustls"]
+native-tls = ["dep:native-tls"]
+rustls = ["dep:rustls", "webpki-roots"]
+
+[dependencies]
+tokio = { version = "1.32.0", features = ["rt", "macros"] }
+chrono = "0.4.22"
+lazy_static = "1.4.0"
+
+md5-rs = "0.1.5"
+hex = "0.4.3"
+base64 = "0.13.0"
+
+alvarium-annotator = { git = "https://github.com/DyrellC/alvarium-annotator" }
```

Comment:
@DyrellC Does this reference need to be updated now that the `alvarium-annotator` repo is populated?

Reply:
Yes all of these will be updated to use the appropriate repo's now that https://github.com/project-alvarium/alvarium-annotator/pull/1 has been updated